packages/sdk/src/interfaces/llm.ts

```diff
@@ -18,7 +21,7 @@ const ToolChoiceSchema = z.object({
 
 const MessageSchema = z.object({
   role: z.enum(['user', 'assistant']),
-  type: z.enum(['text', 'tool_calls', 'tool_result', 'multipart']),
+  type: z.enum(['text', 'tool_calls', 'tool_result', 'multipart']).default('text'),
```

Comment:
**nitpick(non-blocking):** I prefer using zui `.placeholder()` feature than `.default()` as:

1. it's not a real default value; if the value is undefined the schema won't set it
2. the typings will mark the inferred type as optional but should not be.

I tried `.placeholder()` last week and it worked in the studio, but I'm aware some zui features might not work well so I made this comment non-blocking

Reply:
Done in https://github.com/botpress/botpress/pull/13159/commits/70ac2a1e075754118fb9fcdf0fd29cfb47851005